test/bench/bench.cpp

```diff
@@ -234,6 +236,17 @@ void load_benchmark(const fs::path& path, const std::string& name_prefix)
 
             case InputsReadingState::FuncName:
                 benchmark_case->func_name = std::move(l);
+                st = InputsReadingState::FuncSignature;
+                break;
+
+            case InputsReadingState::FuncSignature:
```

Comment:
@chfast suggested to conflate the signature into the name line, such as `keccak256_bnech iii:i`

Reply:
That would be nicer for reading input files, but I don't insist.